Jenkinsfile

```diff
@@ -1,5 +1,4 @@
-buildPlugin(configurations: [
-    [platform: 'linux', jdk: 8],
-    [platform: 'linux', jdk: 11],
-    [platform: 'windows', jdk: 8],
+buildPlugin(useContainerAgent: true, configurations: [
+    [platform: 'linux', jdk: 17],
+    [platform: 'windows', jdk: 11],
```

Comment:
if no more java 8 support maybe worth to add the minimum jdk requirement in the PR title as is this will be in the release notes

Reply:
Thanks. I generally don't include minimum core version changes in release notes for the plugins I maintain (unless the minimum core version is unusual, like a weekly release or a very recent LTS) because such plugin releases aren't even advertised in the UI to users of older cores. Similarly, a recent core version implies a minimum Java version, and users running older Java versions (and therefore older cores) wouldn't get an advertisement for such a plugin upgrade in the UI, so I generally don't include minimum Java version changes in release notes. I have filed dozens (hundreds?) of PRs without this information in the title and shipped dozens of releases without this information in the release notes, all without any complaints from users or other developers. But I acknowledge that as the plugin maintainer, you should be free to include (or not include) any information you want in the release notes. I hope that this discussion about writing release notes has not been a significant distraction from the comparatively far more laborious task of modernizing the plugin itself.